src/ert/_c_wrappers/job_queue/ert_script.py

```diff
@@ -77,26 +75,25 @@ def initializeAndRun(
         try:
             return self.run(*arguments)
         except AttributeError as e:
+            error_msg = e
             if not hasattr(self, "run"):
-                self.__failed = True
-                return (
-                    f"Script '{self.__class__.__name__}' "
```

Comment:
Cleanup is a bit unspecsific in the commit title. Could you add a message to the body describing a bit more what happens?

Reply:
Moved changes to a separate PR https://github.com/equinor/ert/pull/4831 and added more info to the commit message